context/app/static/js/components/Search/SearchWrapper.jsx

```diff
@@ -264,3 +290,4 @@ SearchWrapper.defaultProps = {
 };
 
 export default SearchWrapper;
+export { getByPath, getOrder }; // For tests
```

Comment:
This file is pretty long. Would it make sense to organize into multiple files?

Reply:
sure!